core/server/data/migrations/versions/5.0/2022-05-21-00-00-regenerate-posts-html.js

```diff
@@ -0,0 +1,70 @@
+const logging = require('@tryghost/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const mobiledocLib = require('../../../../lib/mobiledoc');
+const htmlToPlaintext = require('../../../../../shared/html-to-plaintext');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    logging.info(`Starting regeneration of posts HTML`);
+
+    await knex.transaction(async (trx) => {
+        // get list of posts ids, use .forUpdate to lock rows until the transaction is finished
+        const postIdRows = await knex('posts')
+            .transacting(trx)
+            .forUpdate()
+            .select('id')
+            .where('mobiledoc', 'like', '%["video%')
+            .orWhere('mobiledoc', 'like', '%["product%');
```

Comment:
Was the performance of this query acceptable considering it's not indexed in any way?

Reply:
I tested it on a DB with ~71k posts and it took 4.65s, returning 7 matching posts. I think this is good enough for me?

Reply:
👍 cool